library/cirros

```diff
@@ -1,4 +1,5 @@
 # maintainer: Eric Windisch <ewindisch@docker.com> (@ewindisch)
 
-latest: git://github.com/ewindisch/docker-cirros@1cded459668e8b9dbf4ef976c94c05add9bbd8e9
+latest: git://github.com/ewindisch/docker-cirros@5ef3f5024b0aa80553cc34be9eff6685cb31b458
+0.3.3: git://github.com/ewindisch/docker-cirros@5ef3f5024b0aa80553cc34be9eff6685cb31b458
 0.3.0: git://github.com/ewindisch/docker-cirros@1cded459668e8b9dbf4ef976c94c05add9bbd8e9
```

Comment:
Is there a reason for keeping the old version, 0.3.0, around?  Is it going to be a [supported version](https://registry.hub.docker.com/_/cirros/)?